src/System.Text.Formatting/System/Text/Formatting/CompositeFormat.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Diagnostics;
+using System.Text.Primitive;
```

Comment:
I think we don't need a new namespace. You should use the following namespace names:
System.Text.Formatting
System.Text.Parsing
System.Text for common